CryptKeeperZK/crypt-keeper-extension
# chore: optimize build with webpack string replace plugin

## Explanation

Get rid of clean scripts and replace it with webpack plugin.

## More Information

Fixes #107 

## Screenshots/Screencaps

N/A

## Manual Testing Steps

Just run `npm run dev` or `npm run build` and check if extension loads without errors.

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
